.JuliaFormatter.toml

```diff
@@ -0,0 +1,3 @@
+style = "sciml"
+ignore = ["src/equations/shallow_water_two_layer_1d.jl",
+          "src/equations/shallow_water_two_layer_2d.jl"]
```

Comment:
We should check what's going on there

Reply:
It's this piece of code that trips up the formatter (and myself 🤣):
https://github.com/trixi-framework/Trixi.jl/blob/a62f6873f850d410069abefac869b415c8e5758f/src/equations/shallow_water_two_layer_1d.jl#L419-L442

Unfortunately, adding `#! format: off` before it is not sufficient to make the parser not crap out 😢